src/bank00.asm

```diff
@@ -283,7 +283,7 @@ runMainInputHandler_trampoline:
 call_00_0232:
     jp_to_bank 01, call_01_48be                        ;; 00:0232 $f5 $3e $01 $c3 $d7 $1e
 
-processPhysicsForPlayer_1_trampoline:
```

Comment:
I intentionally called this `_1_` as it goes trough bank 1 to find the pointer. It's very silly how this is done, but I figured it makes more sense to call this `_1_` in line with the other useless trampolines.